docs/docs/security.md

```diff
@@ -1,167 +1,162 @@
+reveal secrets.
+
 Here is a non-exhaustive set of examples:
 
-* **Message sizes**. If your enclave is known to process only two kinds of message of around 1 kilobyte or 100 kilobytes
-  respectively, then the size of the encrypted message by itself leaks information about what kind of message it is.
-* **Message processing time**. The same as message sizes but with processing time, e.g. a message type that takes 1 millisecond 
-  to process vs 100 milliseconds can leak what kind of message it is by simply observing how much work the enclave does when
-  the host passes it the new data.
-* **Storage access patterns**. If your enclave doesn't access the database when processing a message of type A, but does
-  when processing a message of type B, or accesses the database with a different sequence, number or type of accesses,
-  the host can learn the message type by observing those accesses.  
+* **Message sizes**. If an enclave is known to process only two kinds of message of around 1 kilobyte or 100 
+  kilobytes, then the size of the encrypted message by itself leaks information about what kind of message it is.
+* **Message processing time**. If an enclave processes a message type in 1 millisecond and another message type in 
+  100 milliseconds, observing how much work the enclave does for a new message, can reveal what kind of message it is.
+* **Storage access patterns**. Suppose an enclave doesn't access the database when processing a message of type A, 
+  but does when processing a message of type B, or accesses the database with a different sequence, number or type 
+  of accesses. In that case, the host can learn the message type by observing those accesses.
 
-Many of these techniques can be used to reveal fine-grained information, not just message types. For instance if an
-encrypted piece of data contains a number that's then used to control a loop that does data lookups, counting the
-number of external data lookups reveals the number.
+Malicious actors can use these techniques to reveal fine-grained information as well. For instance, if an encrypted
+piece of data contains a number used to control a loop that does data lookups, counting the number of external data
+lookups reveals the number.
 
-Some kinds of architectural side channel attacks can be mitigated by Conclave for you, for instance, messages can be
-padded so all encrypted messages look the same size. This sort of approach works if message sizes don't vary too wildly
-and you can afford to use the bandwidth and storage to set all messages to their maximum possible size. How valuable it
-is in your situation is a topic for you to consider as you design your app.
+Conclave can mitigate some of these architectural side-channel attacks. For instance, Conclave can pad all messages 
+to make all encrypted messages appear the same size. This approach works if message sizes don't vary wildly and you 
```

Comment:
Would it make more sense to mention this mitigation that Conclave provides in the `Message sizes` bullet point above?

Reply:
Done.